case_studies/san_leandro_high_school/01_club_fair/planning.md

```diff
@@ -1 +1,22 @@
 # Planning
+
+##### Plans for Event:
+  1. Club table
+    * Hack club banner/poster
+    * Explaining our mission/what we do.
+  2. Sign-ups
+    * 2 sheets
+    * Name, email, phone #
+    * Will be transferred to Google Doc afterward
+  3. Flyers to pass out
+    * Approximately 4 in * 4 in
+    * Logo, meeting times, contact info
+    
+##### Problems Anticipated:
+  1. Misunderstandings of what we do
+    * Programming, not illegal hacking
+  2. Illegible information on sign-up sheet
+    * Unable to contact potential members
+    
+##### Other Associated Planning:
+  1. Other clubs/SLHS administration planning club day→ large event
```

Comment:
Same comments for this section as in https://github.com/hackedu/hackedu/pull/206/files#r39449571.